### PR TITLE
[Docker step0] ws image for Prod

### PR DESCRIPTION
**Pull request by dvdn**
This PR adds docker configs to create Chaos image for production

relates to BOT-2106

You cannot test this image locally

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -1,40 +1,19 @@
...
-      - DOMAIN_NAME=chaos-ws.local.canaltp.fr
-      - CACHE_TYPE=redis
-      - CACHE_REDIS_HOST=redis
-      - NAVITIA_CACHE_TIMEOUT=86400
-      - NAVITIA_PUBDATE_CACHE_TIMEOUT=10
-    command: /bin/bash /var/www/Chaos/docker/honcho_launcher.sh
+      - RABBITMQ_ENABLED=1
+      - RABBITMQ_CONNECTION_STRING=pyamqp://guest:guest@172.21.0.1:5672//?heartbeat=60
+      - NAVITIA_URL=http://navitia2-ws.ctp.dev.canaltp.fr
+      - SQLALCHEMY_DATABASE_URI=postgresql://navitia:navitia@172.21.0.1/chaos
```
Comment (pthegner) [inline docker-compose.yml:13]: what is 172.21.0.1 ?

File: docker-compose.yml [hunk 2]
```diff
@@ -1,40 +1,19 @@
...
-      - SQLALCHEMY_DATABASE_URI=postgresql://navitia:navitia@database/chaos
-      - RABBITMQ_CONNECTION_STRING=pyamqp://guest:guest@rabbitmq:5672//?heartbeat=60
-      - NAVITIA_URL=http://api.navitia.io
-      - DOMAIN_NAME=chaos-ws.local.canaltp.fr
-      - CACHE_TYPE=redis
-      - CACHE_REDIS_HOST=redis
-      - NAVITIA_CACHE_TIMEOUT=86400
-      - NAVITIA_PUBDATE_CACHE_TIMEOUT=10
-    command: /bin/bash /var/www/Chaos/docker/honcho_launcher.sh
+      - RABBITMQ_ENABLED=1
```
Comment (pthegner) [inline docker-compose.yml:10]: Not necessary (default 1)

File: docker-compose.yml [hunk 3]
```diff
@@ -1,40 +1,19 @@
...
+      - RABBITMQ_CONNECTION_STRING=pyamqp://guest:guest@172.21.0.1:5672//?heartbeat=60
+      - NAVITIA_URL=http://navitia2-ws.ctp.dev.canaltp.fr
```
Comment (pthegner) [inline docker-compose.yml:12]: Not necessary (default http://navitia2-ws.ctp.dev.canaltp.fr)

File: docker-compose.yml (hunk 1 above)
Comment (pthegner) [reply docker-compose.yml:13]: see https://github.com/CanalTP/datahub/blob/master/docker-compose.yml for reference -> use environment variables for the right side

File: docker-compose.yml [hunk 4]
```diff
@@ -1,40 +1,19 @@
-version: '2'
+version: '3.8'
 
 services:
   ws:
-    image: navitia/chaos-ws:dev
+    image: chaos-prod:1.0.0
```
Comment (Netmisa) [inline docker-compose.yml:5]: Why `chaos-prod` ?
Image are already on the official docker hub. https://hub.docker.com/r/navitia/chaos-ws 

```suggestion
    image: navitia/chaos-ws:${VERSION}
```

I think version must be a variable for devops and deploy.

File: migrations/versions/6b24727c1bb_clean_line_section.py [hunk 5]
```diff
@@ -0,0 +1,32 @@
+"""clean line_section
```
Comment (Fahmus) [inline migrations/versions/6b24727c1bb_clean_line_section.py:1]: Rebase is needed?

File: docker-compose.yml [hunk 6]
```diff
@@ -1,42 +1,57 @@
-version: '2'
+version: '3.8'
 
 services:
   ws:
-    image: navitia/chaos-ws:dev
+    image: navitia/chaos-ws:${VERSION}
```
Comment (pthegner) [inline docker-compose.yml:5]:
```suggestion
    image: ${REGISTRY_HOST}/navitia/chaos-ws:${VERSION}
```

Comment (pthegner) [reply docker-compose.yml:5]: If registry_host not necessary, you should delete it from docker/build.sh (maybe it's "navitia" that is the registry ?)

Comment (dvdn) [reply docker-compose.yml:5]: :smile: this remark is sooo PR #446

Comment (pthegner) [reply docker-compose.yml:5]: horror

**Review by Fahmus — APPROVED** [review]
Tech ok!

**Pull request merged**